requirements-testing.txt

```diff
@@ -1,4 +1,8 @@
 pytest>=7.2,<8.0
 pytest-runner
 pytest-cov
-codecov
\ No newline at end of file
+codecov
+flake8>=2.1.0
+nbval
+scipy
+cloudpickle
```

Comment:
I think that Cloudpickle is a hard requirement at this point because native Python pickle does not work with Devito objects.

Reply:
ok